openassessment/fileupload/backends/base.py

```diff
@@ -123,7 +123,7 @@ def _get_key_name(self, key):
         Returns:
             A key name (str) to use constructing URLs.
         """
-        return "{prefix}/{key}".format(
+        return u"{prefix}/{key}".format(
```

Comment:
That looks like it might have been a mistake.  I'm surprised this didn't cause failures in the current test suite, because I'm pretty sure we test with keys that have non-ASCII values.